Modules/jks/jks-utils.env

```diff
@@ -104,9 +98,15 @@ _t4dDebugInfoStart "_t4dModJksUtilsStart" "$#" "$@"
         Empty)
             _t4dDebugLog $pwarning "Build #1, Default config will be launched"
             if [[ "$_ExtraArgParam" != "" ]]; then
```

Comment:
BEFORE : BuildWithParams was not available unless we already launched a build
AFTER : it will do it automatically